### ISSUE TITLE
Can you or someone give me html page where you backup config

### ISSUE THREAD

**Issue by vido89** [logon84/Hacking-Huawei-Hg8012H-ONT#14]
Hi all,
I have similar router bat with just regular user account and I cant backup my config
so can you open this url https://192.168.100.1/html/management/cfgfileroot.asp and save that
html page and paste it on baste bin or somehow attach it to this post ?

Thank you Slobodan

**Comment by pmgcosta** [comment]
That page isnt available.

You want the content of hw_ctree.xml file ?

**Comment by vido89** [comment]
Yes to me its not available as Im logged in as regular user, but for telecomadmin user it is available 
Yes I want  hw_ctree.xml file
Tnx Slobodan

Edit: Ah you mean on your router that page does not exists ?

**Comment by pmgcosta** [comment]
Enjoy

[hw_ctree.zip](https://github.com/logon84/Hacking_Huawei_HG8012H_ONT/files/5890369/hw_ctree.zip)

**Comment by vido89** [comment]
@pmgcosta I dont need backup hw_ctree.xml file I need html page from which you downloaded .xml

**Comment by pmgcosta** [comment]
Result: [page.zip](https://github.com/logon84/Hacking_Huawei_HG8012H_ONT/files/5893703/page.zip)

https://pastebin.com/raw/UG6SxUMX

![image](https://user-images.githubusercontent.com/5082655/106273448-9c939c00-622a-11eb-86bd-32c7e4f7c7d8.png)

**Comment by vido89** [comment]
Yes thats it but it does not match my HG8245H it a bit different, tnx
